app/actions/remote/post.ts

```diff
@@ -290,6 +289,9 @@ export const fetchPostsForCurrentChannel = async (serverUrl: string) => {
 
 export async function fetchPostsForChannel(serverUrl: string, channelId: string, fetchOnly = false) {
     try {
+        if (!fetchOnly) {
```

Comment:
I move the `loading messages` handling to all different `fetchPosts` functions, since makes the experience better.
Also, we manage this only if we are not only fetching, mainly to deal with this function that calls on lines 300 and 303 fetch posts or fetchpostssince.

For example, now when we enter a channel and we are still loading from the navigation, it shows.